README.md

```diff
@@ -3,58 +3,54 @@
-
-These should be copied using a bash shell as follows:
+You can read more about creating these environments in this article: [Creating Reproducible Development Environments](https://johnrofrano.medium.com/creating-reproducible-development-environments-fac8d6471f35)
+## Setup
+You should clone this repo, change into the repo directory and open Visual Studio Code using the `code .` command. VS Code will prompt you to reopen in a container and you should say **yes**. This will take a while as it builds the Docker image and creates a container from it to develop in.
 
 ```bash
-    cp .gitignore  ../<your_repo_folder>/
-    cp .flaskenv ../<your_repo_folder>/
-    cp .gitattributes ../<your_repo_folder>/
+git clone https://github.com/CSCI-GA-2820-SP23-003/shopcarts.git
+cd shopcarts
+code .
 ```
+Note that there is a period `.` after the `code` command. This tells Visual Studio Code to open the editor and load the current folder of files.
+
+Once the environment is loaded you should be placed at a `bash` prompt in the `/app` folder inside of the development container. This folder is mounted to the current working directory of your repository on your computer. This means that any file you edit while inside of the `/app` folder in the container is actually being edited on your computer. You can then commit your changes to `git` from either inside or outside of the container.
 
-## Contents
-
-The project contains the following:
-
-```text
-.gitignore          - this will ignore vagrant and other metadata files
-.flaskenv           - Environment variables to configure Flask
-.gitattributes      - File to gix Windows CRLF issues
-.devcontainers/     - Folder with support for VSCode Remote Containers
-dot-env-example     - copy to .env to use environment variables
-requirements.txt    - list if Python libraries required by your code
-config.py           - configuration parameters
-
-service/                   - service python package
-├── __init__.py            - package initializer
-├── models.py              - module with business models
-├── routes.py              - module with service routes
-└── common                 - common code package
-    ├── error_handlers.py  - HTTP error handling code
-    ├── log_handlers.py    - logging setup code
-    └── status.py          - HTTP status constants
-
-tests/              - test cases package
-├── __init__.py     - package initializer
-├── test_models.py  - test suite for business models
-└── test_routes.py  - test suite for service routes
+You can run test on this folder.
+```bash
+cd shopcarts
+nosetests -v --with-spec --spec-color
 ```
 
+## APIS
+
+| URL | Description | Return code |
+| -------- | -------- | -------- |
+| GET /shopcarts/ | Return all REST API name, all available paths | 200 |
+| GET /shopcarts/<int:customer_id> | Return all the items in customer<customer_id> shopcart lists| |
```

Comment:
isnt the API call for this `GET /shopcarts/<int:customer_id>/items` ?

Reply:
Yes